README.md

```diff
@@ -305,7 +305,10 @@ These are the available config options for making requests. Only the `url` is re
   // 'proxy' defines the hostname and port of the proxy server
   proxy: {
     host: '127.0.0.1',
-    port: 9000
+    port: 9000,
+    // `auth` indicates that HTTP Basic auth should be used to connect to the proxy, and supplies credentials.
+    // This will set an `Proxy-Authorization` header, overwriting any existing `Proxy-Authorization` custom headers you have set using `headers`.
+    auth: 'mikeymike:rapunz3l'
```

Comment:
I order to keep a consistent API, please define `auth` as the `auth` parameter used for non-proxy requests.